themes/socialbase/components/05-templates/page-node/page-node.scss

```diff
@@ -5,3 +5,9 @@
     margin-top: 0;
   }
 }
+
+.page-node-type-discussion {
```

Comment:
@agami4 Why do we need specific rule just for discussion here? Can't we make it general? Or fix in discussion module instead?